mindsdb/integrations/handlers/cloud_sql_handler/requirements.txt

```diff
@@ -0,0 +1,4 @@
+psycopg[binary] >= 1.15.3
+docker >= 5.0.3
```

Comment:
Do we need docker as dependency?

Reply:
I added this because the MYSQL handler is using it, Zoran. This makes use of the MYSQL handler.

Reply:
Hmm, strange. I am not sure why we need it, maybe for tests. I will check that. Thanks